packages/playwright/src/index.ts

```diff
@@ -147,7 +147,14 @@ export default class AxeBuilder {
       context
     );
     const partials = await partialResults.getPartials();
-    return this.finishRun(partials);
+
+    try {
+      return await this.finishRun(partials);
+    } catch (error) {
+      throw new Error(
+        `Error: ${error}\n Please check out https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/error-handling.md`
```

Comment:
Nitpick - prefixing the error message with "Error:" is redundant.

```
> throw new Error('Error: blah blah blah\nPlease check out https://......')
Uncaught Error: Error: blah blah blah
Please check out https://......
>
```

This now reads _Uncaught Error: Error_. Perhaps remove the leading "Error:" text, as it's already provided by the JS engine?


```suggestion
        `${error}\n Please check out https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/error-handling.md`
```